manuscript/060_Statement_First_Programming.md

```diff
@@ -135,39 +134,41 @@ var configuration = new XmlConfiguration(customFixture);
 ...
 ```
 
-And it passes -- cool... not. Ok, what is wrong with this? Nothing big, unless you read the source code of XmlConfiguration class carefully. Inside, you can see, how the xml string is stored:
+And it passes -- cool... not. Ok, what is wrong with this? Nothing big, unless you read the source code of XmlConfiguration class carefully. Inside, you can see, how the XML string is stored:
 
 ```csharp
 private static string xmlText; //note the static keyword!
 ```
 
-What the...? Well, well, here is what happened: the author of this class coded in a�small little optimization. He thought: �In this app, the configuration is only modified by members of the support staff and to do it, they have to shut down the system, so, there is no need to read the XML file every time an XmlConfiguration object is created. I�can save some CPU cycles and I/O operations by reading it only once when the first object is created. Another created object will just use the same XML!". Good for him, not so good for you. Why? Because (unless your Statement is evaluated prior to being fulfilled), your custom xml string will never actually be used!
+What the...? Well, well, here is what happened: the author of this class applied a small optimization. He thought: “In this app, the configuration is only modified by members of the support staff and to do it, they have to shut down the system, so, there is no need to read the XML file every time an XmlConfiguration object is created. I can save some CPU cycles and I/O operations by reading it only once when the first object is created. Later objects will just use the same XML!". Good for him, not so good for you. Why? Because your custom XML string will never actually be used! (Unless your Statement is evaluated prior to being fulfilled.)
```

Comment:
Not sure if the exclamation mark should be kept at such a distance! I agree that the parenthetical remark is part of the sentence with "will never actually be used". However these remarks hinder reading of the main line to some extend.

As I noted in the first remark of this PR, I've trouble to understand what
>Unless your Statement is evaluated prior to being fulfilled.

means: i.e. what exactly would make that the XML string be used?
As it's formulated now it's complicating what I thought I understood ;)